service/gcs/runtime/runc/runc.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	runcPath          = "/sbin/runc"
-	containerFilesDir = "/var/lib/gcsrunc"
+	containerFilesDir = "/var/run/gcsrunc"
```

Comment:
lib is stored on the disk (usually sda). run however is stored in tmpfs, so it should work even on readonly initrds.